ui/core/src/config.sandpit.json

```diff
@@ -2,8 +2,43 @@
   "sifAddrPrefix": "sif",
   "sifApiUrl": "https://lcd-sandpit.sifchain.finance/",
   "sifWsUrl": "wss://rpc-sandpit.sifchain.finance/websocket",
+  "sifRpcUrl": "https://rpc-sandpit.sifchain.finance/",
   "sifChainId": "sandpit",
   "web3Provider": "metamask",
   "nativeAsset": "rowan",
-  "bridgebankContractAddress": "0xa5658e85bc7AC7F0388E60E58BEd5aFB41143330"
+  "bridgebankContractAddress": "0xa5658e85bc7AC7F0388E60E58BEd5aFB41143330",
+  "keplrChainConfig": {
+    "chainId": "sifchain",
+    "chainName": "Sifchain",
+    "stakeCurrency": {
+      "coinDenom": "ROWAN",
+      "coinMinimalDenom": "rowan",
+      "coinDecimals": 18
+    },
+    "bip44": {
+      "coinType": 118
+    },
+    "bech32Config": {
+      "bech32PrefixAccAddr": "sif",
+      "bech32PrefixAccPub": "sifpub",
+      "bech32PrefixValAddr": "sifvaloper",
+      "bech32PrefixValPub": "sifvaloperpub",
+      "bech32PrefixConsAddr": "sifvalcons",
+      "bech32PrefixConsPub": "sifvalconspub"
+    },
+    "currencies": [],
+    "feeCurrencies": [
+      {
+        "coinDenom": "ROWAN",
+        "coinMinimalDenom": "rowan",
+        "coinDecimals": 18
+      }
+    ],
+    "coinType": 118,
+    "gasPriceStep": {
+      "low": 0.01,
```

Comment:
With the next deployment we are seeing the min gas price to 0.5 rowan https://sifchain.slack.com/archives/C01ANGGQBQS/p1610580693017600?thread_ts=1610534084.011000&cid=C01ANGGQBQS

Reply:
for the sifchain side that is.

Reply:
Once we get sandpit upgraded we will need to do some testing around the gas fees.